pytest_logger/plugin.py

```diff
@@ -164,13 +187,16 @@ def _refresh_link(source, link_name):
         pass
 
 
-def _make_logsdir(tmpdirhandler, logdirlinks):
+def _make_logsdir_tmpdir(tmpdirhandler):
     logsdir = tmpdirhandler.getbasetemp()
     if logsdir.basename.startswith('popen-gw'):
         logsdir = logsdir.join('..')
     logsdir = logsdir.join('logs').ensure(dir=1)
-    for link in logdirlinks:
-        _refresh_link(str(logsdir), link)
+    return logsdir
+
+
+def _make_logsdir_dir(dstname):
+    logsdir = py.path.local(dstname).ensure(dir=1)
```

Comment:
I suppose local().ensure() wouldn't remove contents of existing directory, so two consecutive test sessions would overwrite their files in this directory. Probably not what we want as new log files would be confused with old log files. I'd prefer this to work exactly like --basetemp, which removes directory contents if there were any.